icons/candy-cane.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11.7 5.8c2.3-2.3 6.1-2.3 8.5 0s2.3 6.1 0 8.5c-.8.8-2 .8-2.8 0-.8-.8-.8-2 0-2.8.8-.8.8-2 0-2.8-.8-.8-2-.8-2.8 0l-9.2 9.8c-.8.8-2 .8-2.8 0-.8-.8-.8-2 0-2.8l9.1-9.9z" />
+  <line x1="6" x2="6" y1="17.8" y2="12" />
+  <line x1="10" x2="10" y1="13.6" y2="7.7" />
+  <line x1="14" x2="14" y1="9.3" y2="4.4" />
+  <line x1="18" x2="18" y1="4.5" y2="10" />
```

Comment:
```suggestion
  <path d="M6.62 20.76a2.24 2.24 0 0 1-4-2l6.5-13.02a6.8 6.8 0 0 1 9.01-2.98 6.81 6.81 0 0 1 3.19 8.55c-.52 1.26-1.82 2.12-3.17 1.44a2.27 2.27 0 0 1-1.03-3.01c.63-1.09.2-2.55-.94-3.12a2.27 2.27 0 0 0-3.06 1.12Z" />
  <path d="m17.13 9.74 4.64-1.86" />
  <path d="m7 20-2-6" />
  <path d="M10 14 8 8" />
  <path d="m13 8-1.61-4.83" />
  <path d="m16.18 6.62 1.95-3.86" />
```

Reply:
<img width="663" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/234059731-fb35ccbc-8615-41f5-81fc-f68bc2e2b7a5.png">

@karsa-mistmere What do you think about this `path` instead? It’s made from perfectly on-grid circles, with points made from rotating them exactly 60°, resulting in an overall shape of exactly 20x20px…? See latest `push` for result…

Reply:
What kind of black magic? 😆 

That's _almost_ perfect, a `8.54 by 14` right triangle, as opposed to the ideal `x by x√3` = `8.54 by 14.79`, but it's such a small difference, that I think pixel perfection is definitely worth it.